.tekton/basic-component-pipeline.yaml

```diff
@@ -356,6 +356,30 @@ spec:
       operator: in
       values: [ "false" ]
 
+  - name: roxctl-image-scan
```

Comment:
My main question is what does the task do: checks things and prints a table without failing the pipeline if anything is found, correct? It's fine to have it as informative task but what is broader vision after?

I have a concern that vulnerabilities appear for two reasons: (1) an engineer (or some bot) added or changed something that creates vulnerability, (2) a new vulnerability was detected in one of components that was already used. While blocking the pipeline for the reason (1) is valid, most often what _we_ see happening is (2). Does it make sense to block the pipeline when "it became vulnerable by itself"?
I think, while we can't differentiate (1) and (2) failing a pipeline for vulnerabilities is counter-productive.

Reply:
Currently I just envision it as an informative task. Our main goal is to regularly execute and dog food our ACS tekton tasks.

Long term I think we should work towards presenting ACS tasks as an alternative to the clair-scan task, which is currently mandatory by the Konflux enterprise contracts.

Reply:
Moved this to Slack: https://redhat-internal.slack.com/archives/C02L9F6KR25/p1716384765267419